src/db/mapping.cr

```diff
@@ -46,7 +46,7 @@ module DB
   # * *type* (required) defines its type. In the example above, *title: String* is a shortcut to *title: {type: String}*.
   # * *nilable* defines if a property can be a `Nil`.
   # * **default**: value to use if the property is missing in the result set, or if it's `null` and `nilable` was not set to `true`. If the default value creates a new instance of an object (for example `[1, 2, 3]` or `SomeObject.new`), a different instance will be used each time a row is parsed.
-  # * *key* defines which column to read from a reusltset. It defaults to the name of the property.
+  # * *key* defines which column to read from a ResultSet. It defaults to the name of the property.
```

Comment:
`` `ResultSet` ``

Reply:
oops. Should the Line above also be `` `ResultSet` ``?

Reply:
Sure, that will make all occurrences of ResultSet a link in the docs.